airflow/models/dagrun.py

```diff
@@ -1538,11 +1538,21 @@ def schedule_tis(
                 and not ti.task.outlets
             ):
                 dummy_ti_ids.append((ti.task_id, ti.map_index))
-            elif ti.task.start_from_trigger is True and ti.task.start_trigger_args is not None:
-                ti.start_date = timezone.utcnow()
-                if ti.state != TaskInstanceState.UP_FOR_RESCHEDULE:
-                    ti.try_number += 1
-                ti.defer_task(exception=None, session=session)
+            # check whether the operator supports start execution from triggerer
```

Comment:
This comment is not very useful. Maybe try to explain what happens if the operator supports starting from trigger, and what happens if not?

Reply:
I just added more details. Please take a look. Thanks!